Provide the correct Solaris curl include directory.

This patch should be send upstream, but needs to be reworked into something
that they will accept.

--- libcurl.pc.in.orig	2014-09-10 12:23:52.455848654 -0700
+++ libcurl.pc.in	2014-09-10 12:24:13.252589181 -0700
@@ -26,7 +26,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@includedir@/curl
 supported_protocols="@SUPPORT_PROTOCOLS@"
 supported_features="@SUPPORT_FEATURES@"
 
